.github/workflows/publish.yml

```diff
@@ -10,49 +9,35 @@ on:
     branches:
       - main
       - master
+    paths:
+      - '.release-plan.json'
 
 concurrency:
   group: publish-${{ github.head_ref || github.ref }}
   cancel-in-progress: true
 
 jobs:
-  check-plan:
-    name: "Check Release Plan"
-    runs-on: ubuntu-latest
-    outputs:
-      command: ${{ steps.check-release.outputs.command }}
-
-    steps:
-      - uses: actions/checkout@v5
-        with:
-          fetch-depth: 0
-          ref: 'master'
-      # This will only cause the `check-plan` job to have a result of `success`
-      # when the .release-plan.json file was changed on the last commit. This
-      # plus the fact that this action only runs on main will be enough of a guard
-      - id: check-release
-        run: if git diff --name-only HEAD HEAD~1 | grep -w -q ".release-plan.json"; then echo "command=release"; fi >> $GITHUB_OUTPUT
-
   publish:
     name: "NPM Publish"
     runs-on: ubuntu-latest
-    needs: check-plan
-    if: needs.check-plan.outputs.command == 'release'
     permissions:
       contents: write
       pull-requests: write
       id-token: write
+      attestations: write
 
     steps:
-      - uses: actions/checkout@v5
+      - uses: actions/checkout@v4
       - uses: pnpm/action-setup@v4
-      - uses: actions/setup-node@v5
+      - uses: actions/setup-node@v4
         with:
           node-version: 18
           # This creates an .npmrc that reads the NODE_AUTH_TOKEN environment variable
           registry-url: 'https://registry.npmjs.org'
+          cache: pnpm
       - run: pnpm install --frozen-lockfile
-      - run: pnpm release-plan publish --provenance
+      - name: Publish to NPM
+        run: NPM_CONFIG_PROVENANCE=true pnpm release-plan publish
```

Comment:
we get provenance now

Reply:
provennance was working successfully as of [12.7.3](https://github.com/ember-cli/eslint-plugin-ember/releases/tag/v12.7.3-eslint-plugin-ember).

Why the need to use `NPM_CONFIG_PROVENANCE=true` instead of just `--provenance` now?

Reply:
Not a question for here, but for the release-plan-setup -- none of these choices are "mine" but just "boilerplate so I don't have to maintain anything"